common/v2/features/SendAssets/components/SendAssetsForm.tsx

```diff
@@ -166,6 +150,105 @@ export default function SendAssetsForm({ txConfig, onComplete }: IStepComponentP
   const [isResolvingName, setIsResolvingDomain] = useState(false); // Used to indicate recipient-address is ENS name that is currently attempting to be resolved.
   const [baseAsset, setBaseAsset] = useState({} as Asset);
   const [resolutionError, setResolutionError] = useState<ResolutionError>();
+  const [selectedAsset, setAsset] = useState({} as Asset);
+
+  const SendAssetsSchema = Yup.object().shape({
```

Comment:
Is it possible to extract the Schema into a separate module? It would help reduce file size and increase readability.

Reply:
Not really, as it depends on state in the component.

Reply:
What I see are calls like ` const account = this.parent.account;` and since `this.parent` is a context provided by `Yup` we should be able to abstract it.
Am I missing something?

Reply:
I use the `selectedAsset` state variable to populate an error string f.x.